packages/trader/src/Modules/SmartChartBeta/Components/LaunchModal/__tests__/launch-modal.spec.tsx

```diff
@@ -0,0 +1,112 @@
+import * as React from 'react';
+import { mockStore } from '@deriv/stores';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import TraderProviders from '../../../../../trader-providers';
+import { LocalStore, TURBOS, WS } from '@deriv/shared';
+import Trade from 'Modules/Trading';
+import { useTraderStore } from 'Stores/useTraderStores';
+import { BrowserRouter } from 'react-router-dom';
+import { ActiveSymbols } from '@deriv/api-types';
+import ReactDOM from 'react-dom';
+
+const default_mocked_store = mockStore({
```

Comment:
If you created a new file (_launch-modal.tsx_) then it will be better to test it separately from t_rade.jsx_. Moreover in this case, you won't need all these mocks of stores and TraderProviders, because you don't use `useTraderStore` and etc in you component

Reply:
@kate-deriv  Actually there is no big functionality being done on the launch-modal.tsx except the click of continue button and that callback is also getting handled in the trade.jsx. (one more the toggling of icon w.r.t theme)
The main functionality is being handled in trade.jsx where the localStorage is getting checked to see if the user has already seen modal , if user is logged in and not showing chart on mobile screen when the launch screen is visible. 

To cover all the functionality I had to test the functionality from trade.jsx .

Reply:
We will be removing the modal in two week anyways .

Reply:
Okay)